docs/rules/ternary-parentheses.md

```diff
@@ -1 +1,37 @@
-# Require Ternary Expressions to have condition enclosed by parentheses (ternary-parentheses)
\ No newline at end of file
+# Require ternary expressions to have condition enclosed by parentheses (ternary-parentheses)
+
+This addresses a stylistic issue. Surrounding the condition in parentheses promotes faster scanning of code and identification of the condition within a ternary expression.
+
+```js
+var qux = (foo) ? bar : baz;
```

Comment:
const everywhere in this file, who uses var anyway?

Reply:
yes, because we dont use var anywhere anymore. either const or let.

Reply:
ok, I updated the test to reflect that as well.

Reply:
very nice, i like (in Borat voice, of course).